app/components/confirmation/funding/view.rb

```diff
@@ -47,11 +47,15 @@ def course_subject_one
       end
 
       def show_bursary_funding?
-        bursary_amount.present?
+        trainee.can_apply_for_bursary?
       end
 
       def bursary_amount
-        @bursary_amount ||= CalculateBursary.for_route_and_subject(trainee.training_route.to_sym, course_subject_one)
+        @bursary_amount ||= if trainee.bursary_tier.present?
+                              CalculateBursary.for_tier(trainee.bursary_tier)
+                            else
+                              CalculateBursary.for_route_and_subject(trainee.training_route.to_sym, course_subject_one)
```

Comment:
Just tidying up! There's a method on trainee for this:

```suggestion
                              trainee.bursary_amount
```